app/models/ems_refresh/save_inventory.rb

```diff
@@ -116,7 +116,7 @@ def save_vms_inventory(ems, hashes, target = nil)
 
         found.save!
         h[:id] = found.id
-        found.send(:clear_association_cache)
+        found.reload
```

Comment:
To summarize discussion we've had elsewhere:

I'm guessing the reason why `reload` decreases memory more than `clear_association_cache` is because it allows objects to no longer be reachable from other objects in the heap.  

The problem is that `clear_association_cache` allows the associations and their associations to be GC'd but doesn't allow the current object's attributes to be GC'd.  These attributes may contain strings values directly from the library (fog, etc.) with their own tree of object references.  Because the original string values are not GC'd, neither can their tree of references.  By reloading, we're forcing rails to create a new attributes hash with values persisted from the database.   The original attributes hash values are then able to be GC'd along with their tree of objects.  See [here](https://github.com/rails/rails/blob/v5.0.0.1/activerecord/lib/active_record/persistence.rb#L432-L445).